vuepress/docs/v7.2/docs/reference/local-tips-and-tricks.md

```diff
@@ -60,7 +60,37 @@ The namespace will be recreated, preserving the images already pulled, so it's u
 
 **Q: What do I do if Entando doesn't fully initialize?** 
 
-**A:** The most common cause of this is a networking problem. See the [Network issues](#network-issues) section below for details. If all else fails, reach out to the Entando team on Slack or in the Forums. 
+**A:** The most common cause is a networking problem. See the [Network issues](#network-issues) section below for details. If all else fails, reach out to the Entando team on Slack. 
+
+**Q: What do I do if the AppBuilder is lacking its left menu?** 
+
+**A:** The most common cause is a failure of the EntandoApp to initialize. This could be caused by networking issues (see the previous point) or configuration issues related to the database, Keycloak, etc. The `appbuilder-menu-bff-deployment` will not be created until the EntandoApp installation is successful, and without that service the left menu will be unable to load.  
```

Comment:
```suggestion
**A:** The most common cause is a failure of the EntandoApp to initialize. This could be caused by networking issues (see the previous point) or configuration issues related to the database, Keycloak, etc. The `appbuilder-menu-bff-deployment` will not be created until the EntandoApp installation is successful, and without that service, the left menu will be unable to load.  
```